src/main/java/net/comments/ft/Test2.java

```diff
@@ -1,46 +1,46 @@
 package net.comments.ft;
 
+import net.comments.objects.*;
+import net.comments.selenium.CommentsDriver;
 import org.hamcrest.MatcherAssert;
-import org.hamcrest.Matchers;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-
 public class Test2 {
-    private final WebDriver[] driver = new WebDriver[1];
+    private final CommentsDriver driver = new CommentsDriver();
```

Comment:
It'll be better to initialize it in the constructor. All objects creation is one place.